### ISSUE TITLE
Move away from playdoh + funfactory?

### ISSUE THREAD

**Issue by lmorchard** [mozilla/badges.mozilla.org#95]
It seems like playdoh & funfactory are not en vogue anymore? If there's a dead-end approaching, consider jettisoning those dependencies.

Maybe a part of #90, or maybe something to do after #90

**Comment by lmorchard** [comment]
This is done, insofar as all the code from funfactory has now been copypasta'd and adapted into badgus.base. Future work should be done to clean this up and get rid of unnecessary code
